Assets/Imports/Qualisys/QTM-Unity-Realtime-Streaming/Streaming/RTMarkerStream.cs

```diff
@@ -84,6 +92,8 @@ void Update()
                     markers[i].SetActive(true);
                     markers[i].GetComponent<Renderer>().enabled = visibleMarkers;
                     markers[i].transform.localScale = Vector3.one * markerScale;
+                    // added code below :: Rotate around the y-axis TDW 2022-03-14
+                    markers[i].transform.RotateAround(WorldCenter, Vector3.up, 180);
```

Comment:
similarly - how confident are you that this is the right transformation? We've learned a lot since March 2022, and I don't want to copy-past tech debt if we don't need to. 

This smells like a slop move based on a poor understanding of what's going on, so I just want to make sure its inclusion is well considered.